mypy/stubtest.py

```diff
@@ -946,6 +947,8 @@ def _verify_signature(
             else:
                 yield f'runtime does not have argument "{arg}"'
     for arg in sorted(set(runtime.kwonly) - set(stub.kwonly)):
+        if _is_private_parameter(runtime.kwonly[arg]):
```

Comment:
Not sure I fully understand the logic here, but I think we want to still emit the error on line 958 below, and only skip the one on line 960.